LRRServer/lawreadingrobot/models.py

```diff
@@ -0,0 +1,48 @@
+
+import uuid
+
+from sqlalchemy import JSON, Column, DateTime, Text, Uuid
+from sqlalchemy.sql import func
+
+from .database import Base
+
+
+class BaseMixin(object):
+    created_at = Column(DateTime(timezone=True), server_default=func.now())
+    updated_at = Column(DateTime(timezone=True), onupdate=func.now())
+    # Makes sure the columns are added to the end of the table
+    created_at._creation_order = 9998  # type: ignore
+    updated_at._creation_order = 9999  # type: ignore
+
+
+def new_uuid() -> uuid.UUID:
+    return uuid.uuid4()
+
+
+class HouseBill(Base, BaseMixin):
+    __tablename__: str = 'housebill'
+
+    id = Column(Uuid(as_uuid=True),
+                primary_key=True,
+                index=True,
+                unique=True,
+                default=new_uuid
+                )
+    hb_id = Column(Text, index=True)
```

Comment:
Nit: Consider giving this a more descriptive name+comment? Maybe "revision_id", "legis_id" or "external_id"?

I assume this intended to store the "id" field in the CSVs, but I'm not sure exactly what this represents, and it would be good to distinguish it from the internal "id" field above.